ios/Classes/FacebookModule.m

```diff
@@ -195,6 +195,16 @@ -(id)accessToken
     }, YES);
     return token;
 }
+
+-(id)appID
+{
+    __block NSString * appID;
```

Comment:
@hansemannn I copied the `accessToken` implementation.

```
-(id)accessToken
{
    __block NSString * token;
    TiThreadPerformOnMainThread(^{
		token = [[FBSDKAccessToken currentAccessToken] tokenString];
    }, YES);
    return token;
}
```

So, how should I change it?
Sorry, I'm not an ObjC developer :P